cmd/ocm-backplane/login/login.go

```diff
@@ -125,6 +126,11 @@ func init() {
 		"",
 		"Login using JIRA Id",
 	)
+	flags.BoolVar(
+		&args.clusterInfo,
+		"cluster-info",
+		false, "Print cluster information",
```

Comment:
May be making the help text more clear: `Print basic cluster information after login`